node/service/src/block_sealing.rs

```diff
@@ -115,7 +111,7 @@ pub fn frequency_dev_sealing(
 					Poll::Ready(_instant) => {
 						let engine_cmd =
 							sc_consensus_manual_seal::rpc::EngineCommand::SealNewBlock {
-								create_empty: true,
+								create_empty: sealing_create_empty_blocks,
```

Comment:
Hmm... This logs an error every interval-seconds. I wonder if we need to do the check outside of this to prevent the error. Thoughts?

```ERROR tokio-runtime-worker sc_consensus_manual_seal::rpc: Instant Seal encountered an error: Transaction pool is empty, set create_empty to true, if you want to create empty blocks```

Reply:
I had tried that but it was before I had found and fixed the underlying issue. I can revisit it.

Reply:
Fixed